tests/test_loader_doc_strings.py

```diff
@@ -16,65 +16,60 @@
 # along with Galaxy.  If not, see <http://www.apache.org/licenses/>.
 
 import json
+import os
 import pytest
+import shutil
+import tempfile
 from unittest import mock
 
-from galaxy_importer import constants
 from galaxy_importer import loaders
 
 
-ANSIBLE_DOC_OUTPUT = """
-    {"my_sample_module": {
-        "doc": {
-            "description": ["Sample module for testing."],
-            "short_description": "Sample module for testing",
-            "version_added": "2.8",
-            "options": {
-                "exclude": {
-                    "description": ["This is the message to send..."],
-                    "required": "true"
-                },
-                "use_new": {
-                    "description": ["Control is passed..."],
-                    "version_added": "2.7",
-                    "default": "auto"
-                }
-            }
-        },
-        "examples": null,
-        "metadata": null,
-        "return": {
-            "message": {
-                "description": "The output message the sample module generates"
-            },
-            "original_message": {
-                "description": "The original name param that was passed in",
-                "type": "str"
-            }
-        }
-    }}
-"""
-
-
 @pytest.fixture
```

Comment:
Does doc_string_loader() need to be a pytest.fixture?  At first glance, it seems like it could be a regular method and called from the tests.  Is there an advantage to making it a pytest.fixture?